### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dnkolegov**
Add basic info about our goals and services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # Audits from Matter Labs
 
-This repository contains public audit reports.
+Welcome to our repository of Matter Labs Audit Reports!
```
Comment (anton-matterlabs) [inline README.md]:
```suggestion
Welcome to the repository of Matter Labs Audit Reports! 
```

**Review by anton-matterlabs — APPROVED** [review]

**Comment by jcsec-security** [comment]
Moving my comments in slack here. The following could be considered:
- **Project link**: Although it is a nice idea to point directly to the zksync-related doc, that URL is more prone to change in the future. I would consider just linking to the project's site or the doc root.
- **Description**: Maybe add the language? "review of solidity smart contracts". As we start working with other techs in the future it will be helpful. We can even change it to tags, something like: "Smart contract, Solidity" so it is more compact for longer projects "Smart contracts, Solidity, Circuits, Circom, Rust"
- **Repo structure**: I am a fan of using a "reports" directory, so the root just contains the README.md and "Reports" instead of having a growing listing... that eventually moves the README rendering far to the bottom


**Unrelated:** A "banner profile" for the GH account would be nice to introduce the ML audit initiative a bit

**Comment by dnkolegov** [comment]
> Moving my comments in slack here. The following could be considered:
> 
> * **Project link**: Although it is a nice idea to point directly to the zksync-related doc, that URL is more prone to change in the future. I would consider just linking to the project's site or the doc root.
> * **Description**: Maybe add the language? "review of solidity smart contracts". As we start working with other techs in the future it will be helpful. We can even change it to tags, something like: "Smart contract, Solidity" so it is more compact for longer projects "Smart contracts, Solidity, Circuits, Circom, Rust"
> * **Repo structure**: I am a fan of using a "reports" directory, so the root just contains the README.md and "Reports" instead of having a growing listing... that eventually moves the README rendering far to the bottom
> 
> **Unrelated:** A "banner profile" for the GH account would be nice to introduce the ML audit initiative a bit

fixed

**Comment by mikhail-egorov** [comment]
It might be a good idea to add a Contacts section to the README.md so that interested projects know how to reach out to us for audits.

**Comment by jcsec-security** [comment]
@dnkolegov will create a dedicated address and add it to the profile page

**Pull request merged**